### PR TITLE
css and html for the why section

### PR DESCRIPTION
**Pull request by cutlasskiwi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -65,6 +65,24 @@ <h1>Rädda Bina!</h1>
         </div>
 
         <div class="why">
+            <h2 class="why-h2">Varför?</h2>
+            <div class="why-box">
+                <img class="why-img" src="img/desktop/icon-bee1.png" alt="drawn picture of a bee from the top">
```
Comment (postmodernistx) [inline index.html:70]: I believe the `alt` text could be in Swedish, as the other website copy is in Swedish :)

File: index.html [hunk 2]
```diff
@@ -65,6 +65,24 @@ <h1>Rädda Bina!</h1>
         </div>
 
         <div class="why">
```
Comment (postmodernistx) [inline index.html:67]: Could the `why` div be an article? Just to improve semantics somewhat.

File: index.html [hunk 3]
```diff
@@ -65,6 +65,24 @@ <h1>Rädda Bina!</h1>
...
+            </div>
+
+            <div class="why-box">
+                <img class="why-img" src="img/desktop/icon-bee2.png" alt="drawn picture of a bee from it's side">
+                <h3 class="why-h3">Livsviktiga!</h3>
+                <p class="why-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+            </div>
+
+            <div class="why-box">
+                <img class="why-img" src="img/desktop/icon-bee1.png" alt="drawn picture of a bee">
```
Comment (postmodernistx) [inline index.html:82]: Also alt text here :) Screen readers often read "Image/picture of {{alt text}}", so no need to add the "Picture of" part :)

File: style.css [hunk 4]
```diff
@@ -171,9 +215,48 @@ h1 {
...
+        font-size: 75px;
+        width: 100%;
+    }
+
+    .why-img {
+        width: 60%;
+    }
+
+    .why-h3 {
+        font-size: 48px;
```
Comment (postmodernistx) [inline style.css:274]: Font sizes are best set in `rem` or `em` for accessibility :)

Comment (postmodernistx) [reply style.css:274]: If you decide to use Sass (which I recommend, as you can get the output file compressed → smaller file size → better page loading speed & more sustainable energy resource consumption), you can create a function that makes the conversion easier :) Like this:

```
@function rem($px) { @return ($px / 16px) * 1rem; }
// Use like this:
font-size: rem(48px);
```

16px is the default body copy font size for most websites and the browser default.

**Review by postmodernistx — APPROVED** [review]
Looks good, clean & clear! 👍

**Review by Bjunkz — APPROVED** [review]

**Pull request merged**